pkgs/unified_analytics/lib/src/constants.dart

```diff
@@ -70,7 +70,7 @@ const int kLogFileLength = 2500;
 const String kLogFileName = 'dash-analytics.log';
 
 /// The current version of the package, should be in line with pubspec version.
-const String kPackageVersion = '0.1.2';
+const String kPackageVersion = '0.1.3-dev';
```

Comment:
Pre 1.0 release, I believe any change is considered breaking ([semver.org](https://semver.org/#spec-item-4)), so can we not just bump the second number every time there is a change?

Reply:
Any thoughts on how we should handle pre 1.0.0 releases for a package? @devoncarew I'm indifferent to what we do, just want to match what the rest of the dart-lang repos do